wagtail/admin/tests/test_collections_views.py

```diff
@@ -55,7 +55,9 @@ class TestCollectionsIndexViewAsSuperuser(
     def setUp(self):
         self.login()
 
-    def get(self, params={}):
```

Comment:
In this file, there's probably no need to allocate the variable. I think we can just set these to None and that's enough.